Marlin/Conditionals_LCD.h

```diff
@@ -92,7 +101,7 @@
     #endif
   #endif
 
-  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
+  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER) || ENABLED(ANET_FULL_GRAPHICS_LCD)
```

Comment:
It seems like enabling `ANET_FULL_GRAPHICS_LCD` should also enable `REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER`, but just my first impression….